samples/algorithms/database-search/README.md

```diff
@@ -7,18 +7,19 @@ languages:
 products:
 - qdk
 description: "This sample shows how to use Q# to search for a marked item with Grover's algorithm."
-jupyter:
```

Comment:
This will break mybinder.org deployment, but should not matter to the samples browser, as the workflow for stripping metadata removes this key: 

```
---
page_type: sample
languages:
- qsharp
- python
- csharp
products:
- qdk
description: "This sample shows how to use Q# to search for a marked item with Grover's algorithm."
---

# Database Search Sample
```

https://github.com/microsoft/Quantum/blob/%E2%AD%90samples/samples/algorithms/database-search/README.md

Reply:
I suspected something like that.  There are 7 sample files that use the jupyter fields - 2 of them publish and 5 don't.  I'll have to run those by the samples team to see if they can spot something I'm not seeing.